plugins/available/nvm.plugin.bash

```diff
@@ -9,10 +9,18 @@ cite about-plugin
 about-plugin 'node version manager configuration'
 
 export NVM_DIR="${NVM_DIR:-$HOME/.nvm}"
+
+# first check if NVM is managed by brew
+NVM_BREW_PREFIX=""
+if _bash_it_homebrew_check
+then
+  NVM_BREW_PREFIX=$(brew --prefix nvm 2>/dev/null)
+fi
+
 # This loads nvm
-if _bash_it_homebrew_check && [[ -s "${BASH_IT_HOMEBREW_PREFIX}/nvm.sh" ]]
+if [[ -n "$NVM_BREW_PREFIX" && -s "${NVM_BREW_PREFIX}/nvm.sh" ]]
 then
-  source "${BASH_IT_HOMEBREW_PREFIX}/nvm.sh"
+  source "$(brew --prefix nvm)/nvm.sh"
```

Comment:
Since `$NVM_BREW_PREFIX` already contains the result of `$(brew --prefix nvm)`, shouldn't this be:
```bash
source "${NVM_BREW_PREFIX}/nvm.sh"
```

Reply:
fair point, fixed